File: codeforces/mashups/9/C.go
package main

import "fmt"

// C. Treino do BamBam

type Pair struct {
	Value int
	Index int
}

func main() {

	var n int
	max := Pair{Value: 0, Index: 0}

	fmt.Scanf("%d", &n)

	a := make([]int, 3)

	for i := 0; i < n; i++ {
		var ai int
		fmt.Scanf("%d", &ai)

		a[i%3] += ai

		if a[i%3] > max.Value {
			max.Index = i%3
			max.Value = a[i%3]
		}
	}

	//fmt.Println(a[0], a[1], a[2], max.Index, max.Value)

	switch max.Index {
	case 0:
		fmt.Println("chest")
	case 1:
		fmt.Println("biceps")
	case 2:
		fmt.Println("back")
	}

}

File: codeforces/mashups/9/C.cpp
#include <bits/stdc++.h>

using namespace std;

// C. Treino do BamBam

int main() {

	int n = 0;
    pair<int, int> max = {0, 0};

	cin >> n;

    vector<int> a(3);

	for (int i = 0; i < n; i++) {
		int ai;

		cin >> ai;

		a[i%3] += ai;

		if (a[i%3] > max.first) {
            max.first = a[i%3];
			max.second = i%3;
		}
	}

    //cout << max << a[0] << a[1] << a[2] << endl;

	switch (max.second) {
	case 0:
		cout << "chest\n";
        break;
	case 1:
		cout << "biceps\n";
        break;
	case 2:
		cout << "back\n";
        break;
	}

    return 0;
}
